packages/dx-chart-core/src/plugins/zoom-pan/computeds.ts

```diff
@@ -171,3 +173,25 @@ export const detachEvents = (node: any, handlers: EventHandlers) => {
     node.removeEventListener(el, handlers[el]);
   });
 };
+
+/** @internal */
+export const getRect = (
+  interactionWithArguments: Interaction,
+  interactionWithValues: Interaction,
+  initial: Location,
+  current: Location,
+  bbox: BBox,
+) => {
+  const isZoomArgument = checkInteraction(interactionWithArguments, 'none', 'pan');
```

Comment:
Seems that 'none', 'pan' can be moved inside the checkInteraction function

Reply:
https://github.com/DevExpress/devextreme-reactive/pull/2057/files#diff-ed254ffc4e96edd9af97d74765ab4c8cR86 also use `checkInteraction` function.